src/engine/source/indexerconnector/qa/test_data/test_initialize_indexer_connector/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "wazuh-states-vulnerabilities",
+  "name": "wazuh-states-vulnerabilities-$(date)",
```

Comment:
I think we can leave this change out

Reply:
So, I'll revert all the efficacy test changes